libs/dataset-explorer/src/lib/DistributionBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Link, Stack, Text } from "@fluentui/react";
+import {
+  BasicHighChart,
+  IDistributionBalanceMeasures,
+  LabelWithCallout
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { dataBalanceTabStyles } from "./DataBalanceTab.styles";
+import { getDistributionBalanceMeasuresChart } from "./getDistributionBalanceMeasuresChart";
+
+export interface IDistributionBalanceMeasureProps {
+  distributionBalanceMeasures: IDistributionBalanceMeasures;
+}
+
+export class DistributionBalanceMeasuresChart extends React.PureComponent<IDistributionBalanceMeasureProps> {
+  public render(): React.ReactNode {
+    const distributionBalanceMeasures = this.props.distributionBalanceMeasures;
+    if (!distributionBalanceMeasures) {
+      return;
+    }
+
+    const styles = dataBalanceTabStyles();
+    const measuresLocalization =
+      localization.ModelAssessment.DataBalance.DistributionBalanceMeasures;
+
+    return (
+      <Stack tokens={{ childrenGap: "l1" }}>
+        {/* Renders the title and info hover-over */}
+        <Stack.Item>
+          <Stack horizontal>
+            <Stack.Item>
+              <Text variant="large" className={styles.boldText}>
+                {measuresLocalization.Name}
+              </Text>
+            </Stack.Item>
+
+            <Stack.Item className={styles.callout}>
+              <LabelWithCallout
+                label=""
```

Comment:
Was something meant to be here?

Reply:
No, its empty because I was trying to make an empty `LabelWithCallout` so that just the clickable (i) appears. Not sure if `Callout` would work better here, but this was working well for the use case.